kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/allegro/Product.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.good.patterns.challenges.allegro;
+
+public class Product {
+    String product;
```

Comment:
Wydaje mi się, że kiedyś już to omawialiśmy - mało sensowne jest nazywanie pola tak samo, jak klasa w której się znajduje. Tutaj zapewne chodziło Ci o nazwę produktu, więc lepiej użyć `productName`, a jeszcze lepiej - krócej - po prostu `name`.
Chyba, że to jakiś kod, identyfikator itp. - wtedy też konkretniejsza nazwa (np. `code`, `ean`, `id` itp.)

Reply:
+ pole jest `package private`, czemu nie `private final`?